invokeai/frontend/web/public/locales/en.json

```
@@ -1684,6 +1680,8 @@
         "postProcessingMissingModelWarning": "Visit the <LinkComponent>Model Manager</LinkComponent> to install a post-processing (image to image) model.",
         "missingModelsWarning": "Visit the <LinkComponent>Model Manager</LinkComponent> to install the required models:",
         "mainModelDesc": "Main model (SD1.5 or SDXL architecture)",
+        "outputTooLargeShort": "Output is too large to upscale",
+        "outputTooLarge": "Max upscale limit is 10,000x10,000 pixels. Please try a smaller image or decrease your scale selection.",
```

This needs a variable to interpolate.

If we want to spell out the upscale size limit like this (which I think is good), the config setting should probably be `maxUpscaleDimension: 10_000` rather than `maxUpscalePixels: 10_000 * 10_000`.

Then you can do this:
```suggestion
        "outputTooLarge": "Max upscale limit is {{maxUpscaleDimension}}x{{maxUpscaleDimension}} pixels. Please try a smaller image or decrease your scale selection.",
```

And in the component:
```ts
  const otherWarnings = useMemo(() => {
    const _warnings: string[] = [];
    if (isTooLargeToUpscale) {
      // use toLocaleString() to get commas or periods per user's selected locale
      _warnings.push(t('upscaling.outputTooLarge', { maxUpscaleDimension: maxUpscaleDimension.toLocaleString() }));
    }
    return _warnings;
  }, [isTooLargeToUpscale, t]);
```